recipes/r-bbotk/meta.yaml

```
@@ -0,0 +1,91 @@
+{% set version = '0.2.2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-bbotk
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/bbotk_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/bbotk/bbotk_{{ version }}.tar.gz
+  sha256: bb6387752a1d681111c087886f8889d6c93d660b3b11c0a35cb3b3eafd333f1a
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base >=3.6
```

conda-forge anyway currently only builds for R 3.6 and 4.0. Therefore the version constrain can be removed and the linter will be happy.... This applies to all the recipes here... if a R version is not supported by a package, that can be handled via a `skip: True  # ["r_base" == "X.Y"]` entry in the build section. 
```suggestion
    - r-base
```

---

Yes, revert 31b05e006250c9c4ced51781914b60985c62e3fc

---

I did that now, thanks that you had a look! Now the linter is happy again.